assignment-2/power.md

```diff
@@ -35,14 +37,14 @@ According to one report [1] there were inadequacies in contract planning and pro
 
 The office of inspector general Daniel Levinson states in another report [3] that CMS did not alway manage and oversee the contractor performance. The main problem according to this report is the lack of control on the contract deliverables.
 
-The CMS did not always provide adequate contract management and oversight for Federal marketplace contracts. For example a report of the OIG states [http://oig.hhs.gov/oas/reports/region3/31403001.pdf page 4]:
+For example the CMS did not always provide adequate contract management and oversight for Federal marketplace contracts. For example a report of the OIG states [3, page 4]:
 
 <i>“The FFM contract awarded to CGI Federal on September 30, 2011, required the contractor to provide a quality assurance surveillance plan within 45 days of contract award. The plan provides a systematic and structured process for the Government to evaluate the services the contractor will provide, including but not limited to processes, methods, metrics, customer satisfaction surveys, service-level agreements, and operational level agreements. The contractor’s January 2012 status reports showed that the contractor provided its plan on January 19, 2012, more than 2 months after it was required. CMS officials were unable to show us the plan and said that they never received it. The contracting officer’s representative had not questioned the accuracy of the January 2012 status report. " </i>
 
-Not only the IOG states that CMS didn’t handle the project well. Journalists David Perera and Sean Gallagher speculated that CMS was ill-suited to the task of coordinating all the contractors. Social activist and technologist Clay Johnson later said that the federal government had issues come up given that it "leans towards a write-down-all-the-requirements-then-build-to-those-requirements type of methodology" not well suited to current IT[19]
+Not only the IOG states that CMS didn’t handle the project well. Journalists David Perera and Sean Gallagher speculated that CMS was ill-suited to the task of coordinating all the contractors. Social activist and technologist Clay Johnson later said that the federal government had issues come up given that it "leans towards a write-down-all-the-requirements-then-build-to-those-requirements type of methodology" not well suited to current IT [19].
```

Comment:
As we are writing an academic paper, it is better that we avoid contractions: do not instead of don't, did not instead of didn't, etc.